workspace/python/ds-pack/2.1.0-base-python-3.10.13/README.md

```diff
@@ -0,0 +1,74 @@
+# workspace-python-ds-pack-2.1.0-base-python-3.10.13
+This is the Python Data Science pack image made for Workspaces. 
+The image runs a [Jupyterhub](https://jupyter.org/hub) server that has [PyCharm](https://lp.jetbrains.com/projector/) and [VSCode](https://github.com/coder/code-server) pre-installed with all of the essential and most used packages.
+The images also comes pre installed with [peak-sdk](https://docs.peak.ai/sdk/).
```

Comment:
```suggestion
The images also comes pre installed with [peak-sdk:1.0.0](https://docs.peak.ai/sdk/).
```
NIT

Reply:
Hey i've actually mentioned the version in the readme so don't think we should be adding it here.